simson/plastics/plastics_export.py

```diff
@@ -12,95 +19,402 @@ class PlasticsDataExporter(CommonDataExporter):
+        materials = pd.read_csv("data/plastics/input/dimensions/materials.csv", header=None)[
+            0
+        ].tolist()
+        # goods = pd.read_csv("data/plastics/input/dimensions/goods_in_use.csv", header=None)[0].tolist()
+        # print(mfa.flows["wasteexport => tradepool"].values)
+        # print(mfa.flows["tradepool => wasteimport"].values)
+
+        ds = xr.DataArray(
+            eol_data,
+            coords=[years, elements, regions, materials],
+            dims=["year", "elements", "region", "material"],
+        )
+
+        # 转为 DataFrame，并处理合并和重命名
+        df = ds.to_dataframe(name="EOL").reset_index()
+        df_grouped = df.groupby(["year", "region"], as_index=False)["EOL"].sum()
+        df_grouped.columns = [
+            col.capitalize() if col != "EOL" else col for col in df_grouped.columns
+        ]  # 可选：统一列名风格
+        print(df_grouped[(df_grouped["Region"] == "CHA") & (df_grouped["Year"] == 2019)])
+        print(df_grouped[(df_grouped["Region"] == "USA") & (df_grouped["Year"] == 2019)])
+        print(df_grouped[(df_grouped["Region"] == "EUR") & (df_grouped["Year"] == 2019)])
+
+        # 输出为 CSV
+        df_grouped.to_csv(output_path, index=False)
+        print(f"EOL data exported to {output_path}")
+
+    def export_use_data_by_region_and_year(
+        self, mfa: fd.MFASystem, output_path: str = "use_by_region_year.csv"
+    ):
+        # 假设 "eol" 是 flow 的 key
+        if "fabrication => use" not in mfa.flows:
+            raise KeyError("The MFA system does not contain 'use' in flows.")
+
+        use_data = mfa.flows["fabrication => use"].values  # xarray.DataArray
+        print(use_data.shape)
+        # 转换为 DataFrame 并重命名列
+        years = pd.read_csv("data/plastics/input/dimensions/time_in_years.csv", header=None)[
+            0
+        ].tolist()
+        elements = pd.read_csv("data/plastics/input/dimensions/elements.csv", header=None)[
+            0
+        ].tolist()
+        regions = pd.read_csv("data/plastics/input/dimensions/regions.csv", header=None)[0].tolist()
+        materials = pd.read_csv("data/plastics/input/dimensions/materials.csv", header=None)[
+            0
+        ].tolist()
+        goods = pd.read_csv("data/plastics/input/dimensions/goods_in_use.csv", header=None)[
+            0
+        ].tolist()
+
+        ds = xr.DataArray(
+            use_data,
+            coords=[years, elements, regions, materials, goods],
+            dims=["year", "elements", "region", "material", "goods"],
+        )
+
+        # 转为 DataFrame，并处理合并和重命名
+        df = ds.to_dataframe(name="Use").reset_index()
+        df_grouped = df.groupby(["year", "region"], as_index=False)["Use"].sum()
+        df_grouped.columns = [
+            col.capitalize() if col != "Use" else col for col in df_grouped.columns
+        ]  # 可选：统一列名风格
+
+        # 输出为 CSV
+        df_grouped.to_csv(output_path, index=False)
```

Comment:
```suggestion
        if "use => eol" not in mfa.flows:
            raise KeyError("The MFA system does not contain 'eol' in flows.")

        eol_data = (
            mfa.flows["eol => collected"]
            + mfa.flows["wasteimport => collected"]
            - mfa.flows["collected => wasteexport"]
        )
        df = eol_data.to_df()
        df.to_csv(output_path, index=False)
```

This can be done so much shorter :-D